code/datums/supplypacks/supply.dm

```diff
@@ -179,7 +179,7 @@
 	cost=25
 	containertype = /obj/structure/closet/crate/secure/xion
 	containername = "Explorer equipment"
-	access = access_explorer
+	access = access_mining
 
 /datum/supply_pack/pilotgear
 	name= "Pilot gear"
```

Comment:
I don't think we should remove cataloguers.

Exploration as a content remains. Cataloguers can be used to access black box and other lore bits which will hopefully be engaged with more, not less after the removal.

Along that same line - exploration frequency should remain but that's another comment.

Reply:
Cataloguers are staying, they are being removed from the pilot lockers specifically, and long term we probably are going to move all cataloguers to a single place.

Reply:
lotta miners were useing catalougers as mining vender 2 by ordering it from cargo. TBH I think they are better being moved to science to give some accaul go find shit gameplay instead of just R and D

Reply:
That should be ahelped.